src/xmipp/applications/scripts/deep_consensus/deep_consensus.py

```diff
@@ -24,20 +24,20 @@
  *  e-mail address 'xmipp@cnb.csic.es'
  ***************************************************************************/
 """
-from src.xmipp.bindings.python.xmipp_base import *
+from xmipp_base import *
```

Comment:
`import xmipp_base` (multiple occurrences)? (to be honest, I don't know which one is preferable)

Reply:
The `from src.xmipp.bindings.python.xmipp_base import *`is incorrect for run time executions since the xmipp_base will end in the PYTHONPATH.